server/cloudformation/cloudformation.go

```diff
@@ -339,18 +348,11 @@ func (c *CustomResourceProvisioner) Handle(ctx context.Context, message *sqs.Mes
 	switch err {
 	case nil:
 		resp.Status = StatusSuccess
-		logger.Info(ctx, "cloudformation.provision",
-			"request", req,
-			"response", resp,
-		)
+		logger.Info(ctx, "cloudformation.provision")
```

Comment:
I removed logging the full request and response, since there might be plain text secrets in it. But we should at least log some information about the request/response.